data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/totp_equip_magic_items.lst

```diff
@@ -0,0 +1,33 @@
+# CVS $Revision: 1$ $Author: Axxroy$ -- Sun Feb 26 13:53 2017 -- reformated by prettylst.pl v1.50 (build 22134)
+SOURCELONG:Treasury of the Pharaohs	SOURCESHORT:TotP	SOURCEWEB:http://www.makeyourgamelegendary.com/	SOURCEDATE:2015-04
+
+###BLOCK: Rings
+Necropolis Attendant's Band															TYPE:Magic.Ring	COST:8000		WT:0		MODS:NO			SOURCEPAGE:p.11 	SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=1|Detect Undead																																																																																						SPROP:The wearer can concentrate as a full-round action to detect undead. While the band is worn, unintelligent undead perceive the wearer as not being a threat. As long as the wearer does not engage in any hostile acts, they will be ignored by the undead. An undead creature commanded to attack the wearer ignores this protection.
+Signet of the Life Pharaoh																TYPE:Magic.Ring	COST:45200		WT:0	MODS:NO				SOURCEPAGE:p.11		SPELLS:Magic Item|TIMES=3|CASTERLEVEL=1|Cure Light Wounds	SPELLS:Magic Item|TIMES=1|CASTERLEVEL=5|Cure Serious Wounds		BONUS:COMBAT|AC|2|TYPE=Deflection		SPROP:Ring of protection +2. The wearer can cast cure light wounds three times per day as a standard action, or a cure serious wounds once per day. When you use this ability to heal a good-aligned creature, that creature gains a +5 sacred bonus on attack rolls, skill checks, ability checks, and saving throws for 1 round. Once per week, the spirit of the Life Pharaoh can forestall the death of the wearer. Anytime the wearer would be killed by hit point damage, the ring bestows a breath of life upon her. If this prevents the wearer’s death, her head is surrounded by a beatific halo for up to 1 hour, shedding light as a light spell and enabling her to gain a +10 sacred bonus on a single Wisdom or Charisma check or Wisdom- or Charisma-based skill check. Once this bonus is used, the halo disappears and the signet of the Life Pharaoh cannot use any of its healing abilities for one week.
```

Comment:
Just a very minor nit here: SPELLS:Magic Item| - I prefer to know what item is granting the spell(s). I would prefer Magic Item actually be the name of the item for clarity sake. However, that is just a minor nit and *is* not required for this review.